CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/translators/FactToTableTranslator.java

```diff
@@ -69,7 +70,7 @@ public Object getCellData(final ElementFact cellValue, final String columnName)
         } else if (cellValue.getFactName().equals(columnName)) {
             return cellValue.getFactValue();
         } else {
-            throw new UnrecognisedColumnException("Column not recognised: " + columnName);
+            throw new UnrecognisedColumnException(UNRECOGNISED_COLUMN_EXCEPTION_STRING + columnName);
```

Comment:
This could actually be moved into `AbstractTableTranslator.UnrecognisedColumnException`.

Reply:
Done